md/building-community-edition-from-source.md

```diff
@@ -29,6 +29,6 @@ Note that you can run Bonita BPM on Windows but you cannot build Bonita BPM on W
 
 In order to build Bonita BPM you need to use the script provided in [Build Bonita BPM](https://github.com/Bonitasoft-Community/Build-Bonita-BPM) GitHub project. Use the branch of the project that match the version you want to build.
 
-Simply run `bash build-script-7.3.2.sh` and provide required information.
+Simply run `bash build-script-7.4.1.sh` and provide required information.
```

Comment:
Question, do you update this page every month we release a maintenance version?
This can be cumbersome. Can't we be smarter on this?
For instance, without the third digit, we could use `${varVersion}` instead of hard-coded number.